index.js

```diff
@@ -61,14 +59,52 @@ function dispatcher (hooks) {
     if (hooks.wrapInitialState) initialStateWraps.push(hooks.wrapInitialState)
     if (hooks.wrapReducers) reducerWraps.push(hooks.wrapReducers)
     if (hooks.wrapEffects) effectWraps.push(hooks.wrapEffects)
-    if (hooks.models) hooks.models.forEach(setModel)
+    if (hooks.models) hooks.models.forEach((model) => setModel(model))
```

Comment:
Once again javascript is different than other languages.  I always forget that the arity of the array methods is `element, index, array`, so this kept throwing on line 70 below because the array index is being passed as the second arg to `setModel`, causing it to throw since it's not an object.

anyway, i'm happy to make this a `function` call, realized that you're not using fat arrows in this code base at all